### ISSUE TITLE
Add jumping

### ISSUE THREAD

**Issue by aldopain** [aldopain/cyberpunk#21]
Добавить прыжок.

**Comment by aldopain** [comment]
@fshpsmgc @danibord @DarkSamudra давайте **хорошенько** подумаем, а надо ли оно нам))0)

**Comment by fshpsmgc** [comment]
Давайте. Прыжок в изометрии — отвратительная идея. Можно добавить специальные места, где игрок сможет перепрыгнуть что-то/спрыгнуть/подняться по нажатию контекстной кнопки, если уж вам совсем нужен прыжок, но это тоже такое себе

**Comment by aldopain** [comment]
@fshpsmgc согласен, но, имхо, по контекстной кнопке (та же F) прям ОТЛИЧНО. Думал об этом, так реализовано в Tyranny.

**Comment by DarkSamudra** [comment]
Согласен с такой идеей прыжка. Таким же образом можно будет реализовать залезание в какие-нибудь обходы для стелса

**Comment by danibord** [comment]
Согласен со всем вышесказанным, прикрепление к контексту кнопки действия выглядит вполне нормальной заменой.
